examples/analytics.amp.html

```diff
@@ -186,6 +186,27 @@
 </script>
 </amp-analytics>
 
+<amp-analytics type="piano">
+<script type="application/json">
+{
+  "vars": {
+    "aid": "AIDAIDAID",
+    "contentCreated": "2015-02-16T05:00:00",
+    "contentSection": "Sports",
+    "contentAuthor": "John Higgins",
+    "tags": ["sports"]
+  },
+  "triggers": {
+    "defaultPageview": {
+      "on": "visible",
+      "request": "pageview"
+    }
+  }
+}
+</script>
+</amp-analytics>
+<!-- End Piano example -->
```

Comment:
There doesn't seem to be a start tag for this. Want to add 

```
<!-- Start Piano example -->
```
before the example?